ConsoleApplication/DeltaQuery.cs

```diff
@@ -0,0 +1,230 @@
+﻿// <copyright file="DeltaQuery.cs" company="Microsoft">
```

Comment:
You probably want to have the same copyright header on all files?
probably the MIT one? @jthake-msft ?